src/plugins/vis_types/timeseries/common/types/panel_model.ts

```diff
@@ -162,7 +162,7 @@ export interface Panel {
   max_bars: number;
   pivot_id?: string | Array<string | null>;
   pivot_label?: string;
-  pivot_rows?: string;
+  pivot_rows?: number;
```

Comment:
As long as we don't do a migration, there will be still old `pivot_rows` params in saved object of type string. Either let's keep it as `number | string` or do a migration to be sure it's _actually_ a `number`. Otherwise this can introduce fun bugs later on.